sklearn/feature_extraction/text.py

```diff
@@ -970,6 +976,19 @@ class CountVectorizer(_VectorizerMixin, BaseEstimator):
      [1 0 0 1 1 0 1 1 1]
      [0 1 1 1 0 0 1 0 1]]
 
+
+    >>> vectorizer2 = CountVectorizer(analyzer='word',ngram_range=(2,2))
```

Comment:
this is called `vectorizer2` but the code following this is using `vectorizer`.
also there should be a space after `analyzer='word',`

Reply:
Does it look fine now?